hr_holidays_validity_date/models/hr_holidays.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```

Comment:
simplify the header following https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md